lib/asset/configureItem.js

```diff
@@ -1,39 +1,49 @@
 // Includes
-const generalRequest = require('../util/generalRequest.js').func
+const http = require('../util/http.js').func
+const getGeneralToken = require('../util/getGeneralToken.js').func
 
 // Args
 exports.required = ['id', 'name', 'description']
 exports.optional = ['enableComments', 'sellForRobux', 'genreSelection', 'jar']
 
 // Define
-function configure (jar, id, name, description, enableComments, sellForRobux, genreSelection) {
-  const url = 'https://www.roblox.com/My/Item.aspx?ID=' + id
-  const data = {
-    __EVENTTARGET: 'ctl00$cphRoblox$SubmitButtonTop',
-    ctl00$cphRoblox$DescriptionTextBox: description,
-    ctl00$cphRoblox$NameTextBox: name,
-    ctl00$cphRoblox$actualGenreSelection: genreSelection || 1
-  }
-  if (sellForRobux) {
-    data.ctl00$cphRoblox$SellThisItemCheckBox = 'on'
-    data.ctl00$cphRoblox$SellForRobux = 'on'
-    data.ctl00$cphRoblox$RobuxPrice = sellForRobux
-  }
-  if (enableComments) {
-    data.ctl00$cphRoblox$EnableCommentsCheckBox = 'on'
+function configure (jar, token, id, name, description, enableComments, sellForRobux, genreSelection) {
+  const httpOpt = {
+    url: '//develop.roblox.com/v1/assets/' + id,
+    options: {
+      method: 'PATCH',
+      jar: jar,
+      headers: {
+        'X-CSRF-TOKEN': token
+      },
+      json: {
+        name: name,
+        description: description,
+        enableComments: enableComments,
+        genres: genreSelection || ['All'],
+        isCopyingAllowed: sellForRobux
```

Comment:
I notice this is a boolean in the typings, but the typings says:
```if `sellForRobux` is set it will be put on sale for that amount of robux.```

Your code seems to work with the fact it is a boolean, however - If sellForRobux is true your code will set copyingAllowed to true and there is no mention of a cost. Is this intended behaviour?

Reply:
This value is intended for models and other items that cannot be sold for robux. I can add another parameter that will allow for the setting of price on the other endpoint.

Reply:
I have added the ability to sell products for robux or set them as on sale.